src/main/java/com/google/sps/servlets/InitializeServlet.java

```diff
@@ -26,14 +26,15 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         
-    final String correlator = RequestUtils.getCorrelator(request);
+    final String correlator = RequestUtils.getParameter(request, Property.CORRELATOR);
 
     if (correlator.isEmpty()) {
       response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
       return;
     }
 
     Entity interaction = new Entity(DBUtilities.INTERACTION_TABLE);
+    interaction.setProperty(Property.TIMESTAMP, String.valueOf(System.currentTimeMillis()));
```

Comment:
We can use this timestamp to not only help with calculating the amount of users who interacted on a given day, but also to see how long each session was. This is a common KPI, which can be extremely helpful for advertisers and also shouldn't be that hard to implement with these new additions.